### ISSUE TITLE
Remove Old Clip Accessibility UI

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#10379]
### Bug description

If the User uses the accessibility shortcuts to access clip selection within AU4, deprecated UI will be shown to the User. 

### Steps to reproduce

1. Launch AU4
2. Create a project with track and a clip
3. Tab to track header
4. User right/left arrow key to move tab focus within track header group
5. Press shift to move focus onto first available clip in track 
6. Press Enter
7. Observe old UI 

### Expected behavior

The 'select'/'deselect' button should be removed. 

### Actual behavior

The 'select'/'deselect' button is present 

### Audacity Version

53adc90bd806aa773e48ae9795116c98b628ff9f

### Operating system

OS independent

### Additional context

https://github.com/user-attachments/assets/38123b6f-59d3-417e-8216-ae7bf5c8093a
